### DISCUSSION TITLE
direct cDNA reads contain duplex (fold-back) sequences - intentional?

### DISCUSSION THREAD

**Discussion by brianjohnhaas** [GoekeLab/sg-nex-data#69] [Q&A]
Hi,

In analyzing the direct cDNA sequences, I'm noticing a lot of sequences that have both the forward followed by the reverse complement in the same reads.  Is this intentional, or is it an artifact or otherwise resulting from incomplete processing of the reads?   I don't see this in the direct RNA data, and far less in the other sequencing types - but I'm still exploring it to better understand the overall rates of this phenomenon.

**Comment by brianjohnhaas** [comment]
About 10 to 20% of the direct-cDNA type ONT reads appear to be duplex-type reads, in that the forward read sequence aligns to its reverse complement across most of the read length. This isn't seen in the other cDNA, direct RNA, or stranded-cDNA sequencing at appreciable levels.  Just good to know as it was throwing off some of our downstream analyses that were sensitive to it.

**Comment by cying111** [comment]
Hi @brianjohnhaas ,

It’s possible that this may be related to the technology, such as library preparation, older sequencing kits, or sequencing errors specific to the direct cDNA protocols. But we really appreciate you sharing this observation with us—thank you!
